### PR TITLE
Avoid finalizers

### PR DESCRIPTION
**Pull request by Therzok**

### CODE REVIEW

File: glib/Timeout.cs [hunk 1]
```diff
@@ -28,69 +28,29 @@ namespace GLib {
...
-				Dispose (false);
-			}
-
-			public void Dispose ()
-			{
-				Dispose (true);
-				GC.SuppressFinalize (this);
-			}
-
-			protected virtual void Dispose (bool disposing)
```
Comment (meebey) [inline glib/Timeout.cs]: This is an API breakage for sub-classes of the Timeout class, not sure if anyone is doing that

**Comment by meebey** [comment]
LGTM

File: glib/Timeout.cs (hunk 1 above)
Comment (Therzok) [inline glib/Timeout.cs]: This is an internal class.

Comment (meebey) [inline glib/Timeout.cs]: Ups, sorry for the noise, I haven't scrolled up high enough to see that it is a nested type :)

File: gtk/Widget.custom [hunk 2]
```diff
@@ -29,36 +29,34 @@ protected Widget (GLib.GType gtype) : base(gtype)
...
 }
 
-public override void Destroy ()
-{
-	base.Destroy ();
-}
-
-[DllImport("gtksharpglue-2", CallingConvention=CallingConvention.Cdecl)]
-static extern IntPtr gtksharp_gtk_widget_get_allocation (IntPtr style);
+[DllImport("libgtk-win32-2.0-0.dll ", CallingConvention=CallingConvention.Cdecl)]
```
Comment (alanmcgovern) [inline gtk/Widget.custom]: Extra space!

Comment (Therzok) [inline gtk/Widget.custom]: Ew. I forgot about this, I fixed it in the main PR.

Comment (Therzok) [inline gtk/Widget.custom]: But this is a mono bug

**Comment by Therzok** [comment]
Superseded by #183

**Pull request closed without merging**